### PR TITLE
Collected small fixes and changes

### PR DESCRIPTION
**Pull request by akohlmey**
**Summary**

This pull request collects multiple small changes and fixes that do not warrant a pull request of their own.

**Related Issue(s)**

N/A

**Author(s)**

Axel Kohlmeyer, Temple U

**Licensing**

By submitting this pull request, I agree, that my contribution will be included in LAMMPS and redistributed under either the GNU General Public License version 2 (GPL v2) or the GNU Lesser General Public License version 2.1 (LGPL v2.1).

**Artificial Intelligence (AI) Tools Usage**

By submitting this pull request, I confirm that I did NOT use any AI tools to generate
all or parts of the code and modifications in this pull request.

**Backward Compatibility**

N/A

**Implementation Notes**

- make FFT unittest testers compatible with single precision FFTs
- add pair style `lj96/cut/kk` to improve test coverage of KOKKOS versions of hybrid pair styles
- improve error messages
- update and improve description of running integrated tests with `ctest`
- Linux packaging improvements
- add test for kspace style zero

**Post Submission Checklist**

- [ ] The feature or features in this pull request is complete
- [x] Licensing information is complete
- [x] Corresponding author information is complete
- [x] The source code follows the LAMMPS formatting guidelines
- [ ] Suitable new documentation files and/or updates to the existing docs are included
- [ ] The added/updated documentation is integrated and tested with the documentation build system
- [x] The feature has been verified to work with the conventional build system
- [x] The feature has been verified to work with the CMake based build system
- [x] Suitable tests have been added to the unittest tree.

### CODE REVIEW

**Comment by akohlmey** [comment]
@whatDiracIsCooking FYI.

File: unittest/utils/test_fft3d_kokkos.cpp [hunk 1]
```diff
@@ -59,6 +59,12 @@ using namespace FFTValidation;
 // Verbose output control
 bool verbose = false;
 
+#ifdef FFT_SINGLE
```
Comment (whatDiracIsCooking) [inline unittest/utils/test_fft3d_kokkos.cpp]: nit: both this file and `test_fft3d.cpp` include `fft_test_helpers.h`, so we can put these preprocessor directives in said .h.

Also, for this file, my PR should have used `#include "fft_test_helpers.h"` instead of having the leading `"../utils/"`, sorry about that

File: doc/src/Build_development.rst [hunk 2]
```diff
@@ -273,21 +275,22 @@ Unit tests for force styles
 A large part of LAMMPS are different "styles" for computing non-bonded
```
Comment (akohlmey) [inline doc/src/Build_development.rst:269]: @megmcca here is an update of the force style test docs. Please let me know, if there could be additional improvements.

File: unittest/utils/test_fft3d_kokkos.cpp (hunk 1 above)
Comment (akohlmey) [reply unittest/utils/test_fft3d_kokkos.cpp]: Made the change and some further refactoring, since there were tolerance constants for the two precision settings already provided.

**Review by jmgoff — APPROVED** [review]
The additions and small syntax changes look good.

I assume the adjustments for the tolerances in the `fft_test_helpers` are reasonable, and were mostly changed to work with the updated unit tests for fft3d and fft3d/kokkos.

**Review by jtclemm — APPROVED** [review]
